src/routes/areas/fetchAllAreas.test.ts

```diff
@@ -0,0 +1,19 @@
+import { FastifyInstance, RouteOptions } from "fastify";
+import fetchAllAreas from "./fetchAllAreas";
+import createServer from '../../server';
+describe('GET /area', () => {
+  let server:FastifyInstance;
+  beforeAll(() => {
+    server = createServer({authRoute: [fetchAllAreas]as RouteOptions[]})
+  });
+
+  it('returns a 201 status code when get all areas',async () => {
```

Comment:
Here, we are not validating the data returned, only the status code, so we can create some areas before hitting the endpoint (probably using **beforeAll**) and expect to receive those areas in the response for this specific test.